src/cdk/utils/basic-authorizer-handler/index.ts

```diff
@@ -39,10 +39,27 @@ function createAllowPolicy(
   };
 }
 
+export function getAuthHeaderValue(headers?: {
+  [name: string]: string;
+}): string | undefined {
+  if (!headers) {
+    return undefined;
+  }
+  const headerNames = new Map<string, string>();
+  Object.keys(headers).forEach((name) =>
```

Comment:
I would prefer the usage of `for-of` instead of `forEach`.

Reply:
Changed to `for-of` loop with break/return on first match.